search-app/src/cmr/search/services/query_service.clj

```diff
@@ -16,6 +16,8 @@
    [clj-time.format :as time-format]
    [clojure.set :as set]
    [clojure.string :as string]
+   [clojurewerkz.elastisch.query :as esq]
```

Comment:
With our existing design we do not perform any direct Elasticsearch queries in this namespace.

Reply:
need this for esq/match-all in the filter.